css/components/_figure.scss

```diff
@@ -13,5 +13,10 @@ figure {
     border-top: 1px solid $gray--mid;
     padding: 2px 0;
     font-size: 0.8rem;
+
+    &.thick {
```

Comment:
I'm not a huge fan of this. I personally prefer the normal figcaption styles. At the very least we should probably be consistent on any given page. There are both normal figcaptions and thick ones on this page.